specification/communications-manager.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   version: '3.0'
-  title: Communications Manager API
+  title: NHS Notify API
   description:
     $ref: documentation/APIDescription.md
   contact:
```

Comment:
Not sure if this will be visible, but contact name still references communications manager, do we want to change this to be consistent?

Reply:
Changed it to NHS Notify